src/mame/taito/taito_f3_v.cpp

```diff
@@ -2502,43 +2476,43 @@ inline void taito_f3_state::f3_drawgfx(bitmap_rgb32 &dest_bmp, const rectangle &
 						u32 *dest = dest0;
 						u8 *pri = pri0;
 
-						switch (x)
+						if (x_base > 15)
+						{
+							// pset_o
+							for(int x = x_base & 0xf; x > -1; --x)
+							{
```

Comment:
Inconsistent spacing here, too.

The `if` statement is really just checking bit 4.  It would be clearer as `if (x_base & 0x10)` (or `if (BIT(x_base, 4))`.  The count is the low four bits, so you might want to extract that to a local to make it more explicit.

Also, since you don’t actually use `x` in the loop, would it be clearer to express the loop expression as `(int = 0; x <= x_base & 0xf; ++x)`?